CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Change message log level from warning to debug when environment variables
   set inside a container or by APPTAINERENV have a different value than the
   environment variable on the host.
+- Update to golang.org/x/crypto v0.31.0 to fix CVE-2024-45337
```

Comment:
There's no need for a CHANGELOG entry since the CVE isn't relevant, please remove it.